astropy/modeling/polynomial.py

```diff
@@ -1111,13 +1149,14 @@ def _chebderiv1d(self, x, deg):
 
 
 class Legendre2D(OrthoPolynomialBase):
-    """
-    Legendre 2D polynomial.
+    r"""
+    Bivariate Legendre series.
 
     Defined as:
 
-    .. math:: P_{nm}(x,y) = C_{n_m}  L_n(x ) L_m(y)
+    .. math:: P_{nm}(x,y) = \ssumum_{n,m=0}^{n=d,m=d}C_{nm}  L_n(x ) L_m(y)
```

Comment:
Did you mean "sumsum" as above or is this something else? Also, maybe a squash of all typos is in order?